Rakefile

```diff
@@ -2,5 +2,19 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require File.expand_path('../config/application', __FILE__)
+require "rubocop/rake_task"
+require 'scss_lint/rake_task'
 
 Rails.application.load_tasks
+
+task :default => [:spec, 'jasmine:ci']
+RuboCop::RakeTask.new(:rubocop) do |t|
+  t.patterns = %w(app config test Gemfile)
+end
+
+SCSSLint::RakeTask.new do |t|
+  t.files = Dir.glob(["app/assets/stylesheets"])
+end
+
+task default: [:spec, "jasmine:ci", :rubocop, :scss_lint]
```

Comment:
This line and line 10 won't work together, I don't think.

Also, rubocop-govuk and scss_lint-govuk are dev/test dependencies, and all these Rakefiles run on Production deploys. You'll have to do something like in https://github.com/alphagov/email-alert-api/pull/1045.

Reply:
Ah yeah, I obviously didn't read this closely enough...

I've also added a conditional to avoid the Production environment issue :+1: